docs/contributing/pull-request-guidelines/index.md

```diff
@@ -50,6 +50,20 @@ Use this workflow as a reference when you contribute to Autoware.
 
 #### Example
 
+- There are two types of template. One is [small change template](https://github.com/autowarefoundation/autoware/blob/main/.github/PULL_REQUEST_TEMPLATE/small-change.md) and the other one is [standard change template](https://github.com/autowarefoundation/autoware/blob/main/.github/PULL_REQUEST_TEMPLATE/standard-change.md). Select one based on the following condition.
+
+1. Standard Change:
+
+- Complexity: A standard change involves moderate to high complexity, such as implementing new features, making architectural changes, or addressing multiple interconnected issues. These changes often require a deeper understanding of the codebase and might involve collaboration with other team members.
+- Impact: A standard change has a broader impact on the overall system. It might affect multiple components, alter the system's functionality, or influence performance in a significant way. This category includes minor feature additions and minor bug fixes that, while not necessarily complex, have notable consequences for the system. These changes often require thorough testing and evaluation before they can be merged into the main branch.
+
+2. Small Change:
+
+- Complexity: A small change typically involves low complexity, such as updating documentation, making simple refactoring, or adjusting code style. These changes can usually be understood and reviewed quickly, often without extensive collaboration or deep knowledge of the entire codebase.
+- Impact: A small change has a limited or localized impact on the system. It might affect only a specific component or have minimal implications for the system's overall functionality or performance. Small changes generally do not include minor feature additions or minor bug fixes, as those should be treated as standard changes due to their impact. These small changes can generally be merged more quickly and with less rigorous testing.
+
+#### Steps to use an appropriate pull request template
```

Comment:
Please do not add another section at the same level.

```suggestion
##### Steps to use an appropriate pull request template
```

Reply:
fixed